app/assets/stylesheets/screen-ltr.css

```diff
@@ -1,5 +1,5 @@
 /*
  *= require ltr/common
- *= require bootstrap
+ *= require bootstrap-tooltips
```

Comment:
Presumably a fairly short term goal will be to replace the existing vendored tooltip and dropdown support with something based on the bootstrap gem instead? I assume you've tested that there isn't any issue with mixing the two in the short term...

Reply:
Yes, I would expect to replace the vendored tooltip stuff and just use the upstream version. Currently the tooltips appear to work exactly as before (specifically black text on white background, bootstrap has vice-versa).